sdk/core/AzureCore/Source/Pipeline/Policies/LoggingPolicy.swift

```diff
@@ -72,9 +76,24 @@ public class LoggingPolicy: PipelineStage {
     // MARK: PipelineStage Methods
 
     public func on(request: PipelineRequest, then completion: @escaping OnRequestCompletionHandler) {
-        var returnRequest = request.copy()
```

Comment:
In testing with the demo app, it didn't seem like this was necessary - just updating the context on the provided `request` worked, the response logger was able to get the request start time back out of the context just fine.